### PR TITLE
feat(dli): the queue resource supports setting properties

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

The queue resource supports configuration properities.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/dli TESTARGS='-run TestAccDliQueue_sparkDriver'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dli -v -run TestAccDliQueue_sparkDriver -timeout 360m -parallel 4
=== RUN   TestAccDliQueue_sparkDriver
=== PAUSE TestAccDliQueue_sparkDriver
=== CONT  TestAccDliQueue_sparkDriver
--- PASS: TestAccDliQueue_sparkDriver (289.94s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dli       289.997s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go [hunk 1]
```diff
@@ -382,3 +382,93 @@ resource "huaweicloud_dli_queue" "test" {
...
+// Modify "max_prefetch_instance" parameter, and remove the "max_instance" and "max_concurrent" parameters。
+func testAccDliQueue_sparkDriver_step2(queueName string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_dli_queue" "test" {
+  name     = "%s"
+  cu_count = 64
+
+  spark_driver {
+	max_prefetch_instance = "4"
+  }
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go]:
```suggestion
  spark_driver {
    max_prefetch_instance = "4"
  }
```
invalid format

File: huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go [hunk 2]
```diff
@@ -382,3 +382,93 @@ resource "huaweicloud_dli_queue" "test" {
...
+}`, queueName)
+}
+
+// Remove all properties parameters
+func testAccDliQueue_sparkDriver_step3(queueName string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_dli_queue" "test" {
+  name                       = "%s"
+  cu_count                   = 64
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go]: ditto

File: huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go (hunk 1 above)
Comment (wuzhuanhong) [reply huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go]: fixed

File: huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go (hunk 2 above)
Comment (wuzhuanhong) [reply huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_test.go]: fixed

**Pull request merged**